libraries/Settings/Settings.h

```diff
@@ -5,7 +5,7 @@
 Settings.h is a site-specific set of trike parameters.
 The file SettingsTemplate.h provides examples of how to define parameters,
 and shows the necessary parameters.
-Only Settings.h should be included from sketches.
+Only Settings.h should be included from sketches, but not saved in the git repository.
```

Comment:
Please remove Settings.h from this PR.  Make your changes in SettingsTemplate.h instead.